src/refinement/amr_criteria.cpp

```diff
@@ -37,6 +37,9 @@ AMRFirstDerivative::AMRFirstDerivative(ParameterInput *pin, std::string &block_n
     std::cerr << "Error in " << block_name << ": no field set" << std::endl;
     exit(1);
   }
+  comp6 = pin->GetOrAddInteger(block_name, "comp6", 0);
+  comp5 = pin->GetOrAddInteger(block_name, "comp5", 0);
+  comp4 = pin->GetOrAddInteger(block_name, "comp4", 0);
```

Comment:
I think these names are a little opaque. What about `tensor_component_{6,5,4}`?

Reply:
updated this -- take a look at it now and see what you think.

Reply:
I like it. :+1: